src/components/ids-message/demos/modal-event.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title><%= htmlWebpackPlugin.options.title %></title>
+  <%= htmlWebpackPlugin.options.font %>
+</head>
+<body>
+  <ids-container role="main" padding="8" hidden>
+    <ids-theme-switcher mode="light" version="new"></ids-theme-switcher>
+    <ids-message id="message-example-event" status="info">
+      <ids-text slot="title" font-size="24" type="h2" id="my-message-title">Modal Event Trigger</ids-text>
+      <ids-text id="my-message-contents" align="left">Modal events will be triggered on this message.</ids-text>
+      <ids-modal-button slot="buttons" type="secondary" id="my-message-cancel" cancel>Cancel</ids-modal-button>
+      <ids-modal-button slot="buttons" type="primary" id="my-message-confirm">Confirm</ids-modal-button>
+    </ids-message>
+
+    <ids-layout-grid auto="true">
+      <ids-layout-grid-cell>
+        <ids-text font-size="12" type="h1">Message</ids-text>
```

Comment:
```suggestion
        <ids-text font-size="12" type="h1">Message (events)</ids-text>
```

Also can we rename this example to events.html (not modal directly when its under message)